docs/guides/kipu-optimization.ipynb

```diff
@@ -274,6 +274,16 @@
     "# ruff: noqa: F821"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5cd4ac8",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"note\">\n",
+    "The following code assumes that you have saved your credentials. If you have not, follow the instructions in <LegacyContent>[Set up an IBM Quantum channel](/guides/setup-channel#iqp)</LegacyContent><CloudContent>[Set up your IBM Cloud account](/guides/cloud-setup#cloud-untrusted)</CloudContent> to authenticate with your API token.\n",
```

Comment:
I don't think we normally use a hash for the cloud setup?

```suggestion
    "The following code assumes that you have saved your credentials. If you have not, follow the instructions in <LegacyContent>[Set up an IBM Quantum channel](/guides/setup-channel#iqp)</LegacyContent><CloudContent>[Set up your IBM Cloud account](/guides/cloud-setup)</CloudContent> to authenticate with your API key.\n",
```

Reply:
I think it's helpful to point them to the right spot in the file.